src/schema/inputs/ProjectInput.ts

```diff
@@ -40,8 +40,8 @@ export class UpdateProjectInput {
   @MaxLength(255)
   lifecyclePhase?: string;
 
-  @Field(() => [UpdateNestedTagInput], { nullable: true })
-  tags!: UpdateNestedTagInput[];
+  @Field(() => UpdateNestedTagsetInput, { nullable: true })
+  tagset!: UpdateNestedTagsetInput;
```

Comment:
tagset should be marked as nullable, not as required if the field is set to `nullable: true`